.golangci.yml

```diff
@@ -50,7 +50,6 @@ linters:
     - importas
     - ineffassign
     - intrange
-    - loggercheck
```

Comment:
What's the reason to disable this linter? It does not seem to be enabled by default.

Reply:
I've explained it in the commit message. Here's a copy-paste from the commit:
```
golangci.yml: remove some unused linters
Not sure why they were added in the first place, but from the first
glance they make no sense to be enabled.

 - loggercheck: this repo does not use kitlog, klog, logr, slog, or zap;
 - sloglint: this repo does not use slog;
 - sqlclosecheck: this repo does not use sql;
 - whitespace: probably superseded by gofumpt;
 - zerologlint: this repo does not use zerolog.
```

Reply:
My bad; I totally missed the fact we use klog extensively.

Updated to not remove this linter.